api/lib/certification-session-scheduling/tests/unit/domain/models/Session_test.js

```diff
@@ -0,0 +1,84 @@
+const { expect, sinon } = require('../../../../../../tests/test-helper.js');
+const random = require('../../../../infrastructure/random');
+const { Session } = require('../../../../domain/models/Session');
+const buildSession = require('../../../tooling/buildSession');
+
+describe('Unit | Domain | Models | Session', () => {
+
+  describe('#static schedule ', () => {
+
+    it('should schedule a session if schedule date is in the future in the same timezone', () => {
```

Comment:
**suggestion (non-blocking) :**`if schedule date is in the future in the same timezone` 
C'est quoi le comportement quand la date est déjà passé ou que c'est la date du jour ? J'ai pas trouvé le test dans Session-validation_test.js